server/src/main/java/tabletop/domain/match/tournament/swiss/SwissResultId.java

```diff
@@ -37,20 +38,23 @@ public void setTournamentProcess(TournamentProcess tournamentProcess) {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-
-        SwissResultId that = (SwissResultId) o;
-
-        if (user != null ? !user.equals(that.user) : that.user != null) return false;
-        return tournamentProcess != null ? tournamentProcess.equals(that.tournamentProcess) : that.tournamentProcess == null;
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj instanceof SwissResultId) {
+            SwissResultId entity = (SwissResultId) obj;
+
+            return Objects.equals(getClass(), entity.getClass())
```

Comment:
the getClass() check is not needed here, since we do a instanceof SwissResultId. It is needed in equals/hashCode in IdComparableEntity because it is an abstract class for other entities and they can use these methods.